doc/release-notes.md

```diff
@@ -102,6 +111,27 @@ forwarder:
   linkDialWorkerCount: 10
 ```
 
+## Model Performance Improvements
+Policy relationships are now stored in a denormalized fashion. This means that checking if an entity is tied to another entity via a policy is now a direct lookup, and much faster. This means that the Ziti controller should scale very well in cases where we have many identities, services and/or edge routers. Performance was tested against the APIs used by the SDKs. 
+
+See for more detail:
+
+* [Denormalized Policies](https://github.com/openziti/edge/wiki/Denormalized-Policies)
+* [Characterization (Pure Model Tests)](https://github.com/openziti/ziti/wiki/Characterization#pure-model-tests)
+
+## Data Integrity Checking Framework
+The bbolt datastore used by Ziti provides simple key/value storage with nesting. Ziti has implemented some basic relational constructs on top of bbolt, such as indexed values, foreign key indexes, many to many collections and reference counted collections (for policy denormalization). This release adds a data integrity checking framework which allows us to verify that constraint assumptions are valid, and allows fixing issues when they are found (if possible). This work was done in part to validate that the policy denormalization code is working correctly, and to provide a rememdy if issues are found.
+
+There are two new REST APIs available
+
+* `/database/check-data-integrity`
```

Comment:
Should probably list their supported HTTP methods

Reply:
Done and added links to swagger doc as well